src/DIRAC/FrameworkSystem/Agent/ProxyRenewalAgent.py

```diff
@@ -0,0 +1,85 @@
+"""  ProxyRenewalAgent keeps the proxy repository clean.
```

Comment:
the `ProxyManager` already does something like that, so you may want to remove it from there if we have this agent

Reply:
I think I did. This agent was (re)written because I have moved the purge* actions from ProxyManager to here.